packages/dashboard-frontend/src/components/WorkspaceEditor/index.module.css

```
@@ -0,0 +1,33 @@
+/*
+ * Copyright (c) 2018-2021 Red Hat, Inc.
+ * This program and the accompanying materials are made
+ * available under the terms of the Eclipse Public License 2.0
+ * which is available at https://www.eclipse.org/legal/epl-2.0/
+ *
+ * SPDX-License-Identifier: EPL-2.0
+ *
+ * Contributors:
+ *   Red Hat, Inc. - initial API and implementation
+ */
+
+.workspaceEditor {
+  width: 100%;
+  height: 100%;
+  min-height: 50vh;
+}
+
+.workspaceEditor .monaco {
+  position: relative;
+  width: 100%;
+  height: calc(100% - 38px);
+  min-height: inherit;
+}
+
+.workspaceEditor .monaco .monaco-editor {
+  position: absolute;
+  top: 0;
+  right: 0;
+  bottom: 0;
+  left: 0;
+  border: 1px solid #cecece;
```

The Patternfly guidelines have the recommended [color palette](https://www.patternfly.org/v4/guidelines/colors#color-palette); I'd like you to choose from the colors listed there.

```suggestion
  border: 1px solid var(--pf-global--BorderColor--100);
```

---

Ok. I have fixed it.